README.md

```diff
@@ -114,36 +142,74 @@ At the time of writing, Gym-Ignition offers the following features:
 - Experimental support to create C++ environments
 
+## Demo
+
+We provide two different methods to test Gym-Ignition without the need to install it locally:
+
+1. **Colab notebook**: [![](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/robotology/gym-ignition/blob/master/examples/colab/RandomPolicy.ipynb) and run the example
+1. **Docker image**:
+   ```sh
+   docker pull diegoferigo/gym-ignition:latest
+   pip3 install rocker
+   
+   # Intel GPU
+   rocker --x11 diegoferigo/gym-ignition ./github/examples/python/launch_cartpole.py
+   
+   # Nvidia GPU
+   rocker --x11 --nvidia diegoferigo/gym-ignition ./github/examples/python/launch_cartpole.py
+   ```
+
 ## Setup
 
 The setup instructions expect a **Ubuntu** distribution with at least **Python 3.6**.
+Gym-Ignition is compatible also with other distributions (and, also, other OSs) under the assumption that the Ignition Robotics suite can be installed either from repos or source.
+Though, to keep the instructions simple, we only report the steps for the Ubuntu distro.
 
-The process is different whether you're an _user_ that wants to create environments using Gym-Ignition or you are a _developer_ that wants to edit the Python and C++ code. Common steps:
+The process is different whether you're an _user_ that wants to create environments using Gym-Ignition or you are a _developer_ that wants to edit the Python and C++ upstream code.
 
-1. Install [SWIG](https://github.com/swig/swig) with `apt install swig`
+Execute all the setup commands in the same terminal.
+
+#### Common Steps
 
-### Are you a user?
+1. Install the supported version of **Ignition Gazebo**:
 
-1. Install the Ignition Robotics suite following the [official documentation](https://ignitionrobotics.org/docs/latest/install)
-1. Install Gym-Ignition with `pip install gym-ignition` (preferably in a [virtual environment](https://docs.python.org/3.6/tutorial/venv.html))
-1. Execute the following to export the required environment variables:
    ```sh
-   # Worlds and models path
-   data_path=$(python -c "import gym_ignition_data ; (gym_ignition_data.__path__[0])")
-   export IGN_GAZEBO_RESOURCE_PATH=${data_path}:${data_path}/worlds
-   
-   # Gazebo plugins
-   module_path=$(python -c "import gym_ignition ; print(gym_ignition.__path__[0])" | grep gym_ignition)
-   export IGN_GAZEBO_SYSTEM_PLUGIN_PATH=${module_path}/plugins
+   sudo sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list'
+   wget http://packages.osrfoundation.org/gazebo.key -O - | sudo apt-key add -
+   sudo apt update
+   sudo apt install ignition-gazebo2 libignition-gazebo2-dev
    ```
 
-After these steps, you should be able to execute the example [`launch_cartpole.py`](examples/python/launch_cartpole.py).
+   We currently require the development version, that will be ABI-compatible with the upcoming **Citadel** release.
```

Comment:
I am a bit confused, I thought that Citadel was ignition-gazebo3 .

Reply:
Yes, Citadel is going to be ignition-gazebo3. I checked my local branches and, despite Blueprint has been released before the summer, we still compile against branches ABI-compatible with it.

This check made me realize that we can still use the metapackage `ignition-blueprint` to install everything, and the current instructions actually do not install some (maybe not important) dependency such as `libignition-launch` and libwebsockets.

Reply:
Corrected in 6e0702aea5070dde551be7d78cd2374ab3c863b8